test_pytest_timeout.py

```diff
@@ -16,92 +16,83 @@
 )
 
 
-@pytest.fixture
-def testdir(testdir):
-    if hasattr(testdir, "runpytest_subprocess"):
-        # on pytest-2.8 runpytest runs inline by default
-        # patch the testdir instance to use the subprocess method
-        testdir.runpytest = testdir.runpytest_subprocess
-    return testdir
-
-
-def test_header(testdir):
-    testdir.makepyfile(
+def test_header(pytester):
+    pytester.makepyfile(
         """
         def test_x(): pass
     """
     )
-    result = testdir.runpytest("--timeout=1")
+    result = pytester.runpytest_subprocess("--timeout=0.5")
```

Comment:
I don't remember exactly, but I think this used to use fractional seconds to speed up the tests but did get reverted to 1s.  Probably for flakyness, maybe some other reasons.  Could you leave this to 1s please?

Reply:
Sure